input/docs/cs/noise_filtering.md

```diff
@@ -0,0 +1,72 @@
+title: Filtrování šumu.
+---
+>This document works with two images:
+* [Image of Norway with noise](../media/nor256_noise.png) in `A` 
+* [Image of Norway](../media/nor256.jpg) in `B`
+# Prozkoumání šumu
+``` matlab
+A = imread('nor256_noise.png');
+B = rgb2gray(imread('nor256.jpg'));
+Noise = A ( A~=B );
+imhist(Noise) %funkce zobrazí histogram.
+```
+# Gaussianová konvoluce.
+``` matlab 
+h = fspecial('gaussian',[3 3]); % funkce vytvoří masku pro filtrování 
+Aa = imfilter(A,h,'conv'); %  provádí filtraci obrazových dat
```

Comment:
"provádí filtraci obrazových dat" - to ano, ale to ten řádek pod tím také. Proč jsou tu oba? To chce všechno vysvětlit. Taky Je vhodné sem přidat obrázek výsledku.